packages/framework/azure-client/src/AzureClient.ts

```diff
@@ -40,16 +40,16 @@ export class AzureClient {
     constructor(
         private readonly connectionConfig: AzureConnectionConfig,
         private readonly logger?: ITelemetryBaseLogger,
-        ) {
+    ) {
         this.documentServiceFactory = new RouterliciousDocumentServiceFactory(
             this.connectionConfig.tokenProvider,
         );
     }
 
     /**
-     * Creates a new container instance in the Azure Relay Service.
+     * Creates a new detached container instance in the Azure Relay Service.
      * @param containerSchema - Container schema for the new container.
-     * @returns New container instance along with associated services.
+     * @returns New detached container instance along with associated services.
      */
     public async createContainer(
```

Comment:
This create container code needs changes since it's no longer right. (I would do it but github won't let me comment on part of it.)